IMDbApp/app/src/main/java/com/danielcano/imdbapp/uilayer/viewmodels/MoviesViewModel.kt

```diff
@@ -23,15 +25,29 @@ class MoviesViewModel: ViewModel() {
         loadMovies()
     }
 
-    fun loadMovies(){
+    private fun loadMovies(){
         viewModelScope.launch {
             try{
                 _movieList.value = usecase.getMovies()
-                _status.value = "Successfully loaded"
+                _status.value = "Successfully loaded ${movieList.value!!.size} items"
+                _filteredMovieList.value = _movieList.value
             }catch (e:Exception){
-                _movieList.value = listOf<MovieModel>()
+                _movieList.value = listOf()
                 _status.value = "Failed to load, Error: ${e.localizedMessage}"
```

Comment:
Manejar los textos en recursos

Reply:
Its just a temporal utility I'm using I'm planning on erase it soon. Thanks for the comments.